rpm/osg-pki-tools.spec

```diff
@@ -36,6 +36,8 @@ gzip -c man/osg-incommon-cert-request.1 >%{buildroot}%{_datadir}/man/man1/osg-in
 %{_datadir}/man/man1/osg-incommon-cert-request*
 
 %changelog
+- Improve formatting of osg-incommon-cert-request man page
+
```

Comment:
This change is causing the CI to fail. Can we just remove it and make sure to add it back whenever we cut the next release?

Reply:
A hashtag (that is, comment) seems to make it happy.